### PR TITLE
Eng 1861 clusterip

### PR DESCRIPTION
**Pull request by imosquera**
Adds support using ClusterIP load balancer types

This adds support for continuation of installer as well.

### CODE REVIEW

File: src/install.sh [hunk 1]
```diff
@@ -184,12 +169,15 @@ function get_var() {
...
         export ${var_name}=${default_val}
+        save_response ${var_name}
       fi
     elif [ ! -z "$val_func" ] && ! $val_func ${value}; then
       get_var "$1" $2 $3
     else
       export ${var_name}=${value}
+      save_response ${var_name}
     fi
+    #as vars are set they are saved
```
Comment (kevinawoo) [inline src/install.sh]: what's this comment mean?

File: src/install.sh [hunk 2]
```diff
@@ -514,12 +521,37 @@ function upload_custom_credentials() {
...
+  nginx_pod=$(kubectl $KUBECTL_OPTIONS get pods -o=custom-columns=NAME:.metadata.name | grep nginx | tail -1)
+  cat <<EOL
+
+********************************************************************************
+
+ ClusterIP service type requires port forwarding using 'kubectl port-forward'
+ Please run the following command in another shell:
+
+
+ sudo kubectl $KUBECTL_OPTIONS port-forward $nginx_pod 80:80
```
Comment (kevinawoo) [inline src/install.sh]: sudo?

File: src/install.sh [hunk 3]
```diff
@@ -1166,28 +1200,58 @@ function set_lb_type() {
...
+
+function continue_env() {
+    if [[ ! -z  $NOPROMPT ]]; then
+      return
+    fi
+
+    if [[ -f $CONTINUE_FILE ]]; then
+        get_var "Found continue file at $CONTINUE_FILE from previous run, would you like to use it? (y/n): " USE_CONTINUE_FILE
+        if [[ "$USE_CONTINUE_FILE" == "y" ]]; then
+          source $CONTINUE_FILE
```
Comment (kevinawoo) [inline src/install.sh]: I don't think this works, it'll get sourced in the function, but it won't be available outside the function call.
I had to fix it in my other PR: https://github.com/armory-io/k8s-installer/pull/130/commits/cf4c270f5c0c90db920dfb8ed7586065d9f3851f

File: src/install.sh (hunk 2 above)
Comment (imosquera) [reply src/install.sh]: yah, ideally we use another port but that was going to be harder.

File: src/install.sh (hunk 3 above)
Comment (imosquera) [reply src/install.sh]: This  should work when  tested it.   The vars in file are exported.  Can you test it to confirm?

Comment (kevinawoo) [reply src/install.sh]: ah yeah, seems to work.

**Comment by kevinawoo** [comment]
The continue stuff is missing some variables like 
- bucket name (when i select empty)
- instance size
- gcloud project
- service-account create new one or use existing
- then the service account if selected

File: src/install.sh [hunk 4]
```diff
@@ -1166,28 +1199,58 @@ function set_lb_type() {
...
+  echo "export ${1}=${!1}" >> $CONTINUE_FILE
+}
+
+function continue_env() {
+    if [[ ! -z  $NOPROMPT ]]; then
+      return
+    fi
+
+    if [[ -f $CONTINUE_FILE ]]; then
+        get_var "Found continue file at $CONTINUE_FILE from previous run, would you like to use it? (y/n): " USE_CONTINUE_FILE
```
Comment (kevinawoo) [inline src/install.sh]: I feel like the default should be `y`, since we always prompt after `describe_installer`

Comment (kevinawoo) [reply src/install.sh]: maybe store that `describe_installer` was already prompted and skip the requirement to hit "enter"

**Pull request merged**